combat-trainer.lic

```diff
@@ -2673,12 +2673,7 @@ class TrainerProcess
   end
 
   def pray_mat(game_state)
-    ready_to_mat = (
-      DRCI.exists?(@prayer_mat, @prayer_mat_container) &&
-      DRCI.exists?("wine", @theurgy_supply_container)
-      )
-
-    unless ready_to_mat
+    unless DRCI.exists?(@prayer_mat, @prayer_mat_container)
       DRC.message("*** Cannot run PrayerMat; need prayer mat in prayer_mat_container or in theurgy_supply_container, and wine in theurgy_supply_container")
```

Comment:
Should this error message also be updated?

Reply:
Excellent catch. Thank you!